ROOTFS/etc/waggle/sanity/fatal/core_sdcard.test

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+pf="Recovery SDCard test"
+echo "$pf: Begin"
+
+# run fsck on our sd card
+if [ "$(fsck -n /dev/mmcblk1p1 | grep clean)" = "" ];
+then
+        echo "$pf: /dev/mmcblk1p1 failed fsck FAIL"
+        exit 1
+fi
+
+if [ "$(fsck -n  /dev/mmcblk1p2 | grep clean)" = "" ];
+then
+        echo "$pf: /dev/mmcblk1p2 failed fsck FAIL"
+        exit 2
+fi
+
+mkdir /tmp/core_sdcard_test
+
+mount /dev/mmcblk1p1  /tmp/core_sdcard_test/   
+
+if [ "$(cat /tmp/core_sdcard_test/etc/overlayroot.conf | grep "overlayroot=\"tmpfs")" = ""  ];
+then
+        echo "$pf: sdcard tmpfs not properly set up FAIL"
+        exit 3
+fi
+
+if [ "$?" != "0" ];
+then
+        echo "$pf: Could not run chroot on sdcard, invalid rootfs FAIL"
+        exit 4
+fi
+
+# checking initrd test
+if [ "$(file -L /tmp/core_sdcard_test/boot/initrd | grep "gzip compressed data")" = "" ];
+then
+        echo "$pf: sdcard initrd is not a valid archive FAIL"
+        exit 5
+fi
+
```

Comment:
I would like to add the tests for Image, Image.sig, tegra194-xavier-nx-cti-NGX003-WAGGLE-WS.dtb, tegra194-xavier-nx-cti-NGX003-WAGGLE-WS.dtb.sig on the SD card too